openjdk/openjdk.mk

```diff
@@ -16,6 +16,14 @@ NPROCS:=1
 MEMORY_SIZE:=1024
 
 OS:=$(shell uname -s)
+# Prepend -Xverbosegclog for all operating systems when JDK_IMPL is openj9 or ibm
+ifeq ($(JDK_IMPL),openj9)
+    EXTRA_OPTIONS += -Xverbosegclog
+endif
+
+ifeq ($(JDK_IMPL),ibm)
+    EXTRA_OPTIONS += -Xverbosegclog
+endif
```

Comment:
- please leverage if block at https://github.com/adoptium/aqa-tests/blob/6631872fb637e9f3cd8755b523c8d3d374371654/openjdk/openjdk.mk#L182
- we need to prepend the JVM option